docker-compose.yml

```diff
@@ -68,6 +68,46 @@ services:
       - POSTGRES_USER=${POSTGRES_USER?Variable not set}
       - POSTGRES_DB=${POSTGRES_DB?Variable not set}
 
+  prestart:
+    image: '${DOCKER_IMAGE_BACKEND?Variable not set}:${TAG-latest}'
+    build:
+      context: ./backend
```

Comment:
I think this build context should be `backend/`, if the backend service definition is correct (see below)